<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Beacon</class>
 <widget class="QMainWindow" name="Beacon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Beacon</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabAlertManager">
     <attribute name="title">
      <string>Alert Management</string>
     </attribute>
     <widget class="QPushButton" name="buttonAssign">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Assign</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonReassign">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ReAssign</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableAlerts">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>691</width>
        <height>251</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>a</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tabAgent">
     <attribute name="title">
      <string>Agent</string>
     </attribute>
     <widget class="QPushButton" name="button_StartAgent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Agent</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_StopAgent">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Agent</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupAgentRuntime">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>601</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Agent Runtime</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_AgentPath">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_AgentName">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>40</y>
         <width>281</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Executable</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Args (, seperated)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Args">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>561</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupOutput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>701</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Output</string>
      </property>
      <widget class="QTextBrowser" name="textOutput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>681</width>
         <height>141</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Metric Query</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowserQueryResults">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>701</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEditQuery">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>701</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <widget class="QMenu" name="menuAgent">
      <property name="title">
       <string>Agent</string>
      </property>
      <addaction name="actionExternal_Agent"/>
      <addaction name="actionInternal_Agent"/>
     </widget>
     <addaction name="menuAgent"/>
    </widget>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="menuAdd"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionAlert_Management"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New ...</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load ...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save ...</string>
   </property>
  </action>
  <action name="actionAlert_Management">
   <property name="text">
    <string>Alert Management</string>
   </property>
  </action>
  <action name="actionExternal_Agent">
   <property name="text">
    <string>External Agent</string>
   </property>
   <property name="toolTip">
    <string>Setup and configure a new remote monitoring agent. </string>
   </property>
  </action>
  <action name="actionInternal_Agent">
   <property name="text">
    <string>Internal Agent</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Beacon ...</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login...</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
